game-core/src/main/java/games/strategy/debug/GenericConsole.java

```diff
@@ -1,78 +0,0 @@
-package games.strategy.debug;
-
-import java.awt.BorderLayout;
-import java.awt.Toolkit;
-import java.awt.datatransfer.StringSelection;
-import java.io.PrintStream;
-
-import javax.swing.Action;
-import javax.swing.JFrame;
-import javax.swing.JScrollPane;
-import javax.swing.JTextArea;
-import javax.swing.JToolBar;
-import javax.swing.SwingConstants;
-import javax.swing.SwingUtilities;
-import javax.swing.WindowConstants;
-
-import games.strategy.engine.framework.lookandfeel.LookAndFeelSwingFrameListener;
-import games.strategy.triplea.settings.ClientSetting;
-import games.strategy.ui.SwingAction;
-
-/**
- * Superclass for all debug console windows.
- */
-public abstract class GenericConsole extends JFrame {
```

Comment:
This was renamed to `Console`, perhaps I should have kept the same name for easier diff'ing : |
The changes in this class is we absorb the error console code, and we get a new button for updating the error logging level.